app/views/home/index.html.erb

```diff
@@ -1,3 +1,13 @@
 <% if current_user %>
-  <p><%= link_to 'My Picks', {controller: :picks, action: :index}, method: :get %></p>
+  <p><%= link_to 'My Picks', {controller: :picks, action: :new}, method: :get %></p>
```

Comment:
Why this?

Reply:
It was an artifact from when I was playing around with stuff. I reverted it